test/backwards_compatibility_test.go

```
@@ -0,0 +1,256 @@
+		"--deployment", DEPLOYMENT_ID,
+		"--name", "test-version-channel",
+		"--description", "Test version channel",
+		"--selector", `{"type":"tag","operator":"equals","value":"v1.1.0"}`,
+	)
+	if err != nil {
+		t.Fatalf("Failed to create new deployment version channel: %v", err)
+	}
+	fmt.Println("New deployment version channel output:", newChannelOutput)
+
+	// Skip deployment version channel deletion since endpoint doesn't exist yet
+	runCommand("delete", "deployment-version-channel", "--deployment", DEPLOYMENT_ID, "--name", "test-version-channel")
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)